tools/accuracy_checker/configs/colorization-v2-norebal.yml

```diff
@@ -0,0 +1,68 @@
+evaluations:
+  - name: colorization-v2-norebal
+    module: custom_evaluators.colorization_evaluator.ColorizationEvaluator
+    module_config:
+        network_info:
+          test:
```

Comment:
hm... could you please give more descriptive name for this section? (It is slightly difficult understand which model can be set as test, possible names: `colorizator`, `colorization_net` (than second model can be renamed to `verification_net`), `colorization`

Reply:
@eaidova corrected)